test/test_ion_writer.cpp

```diff
@@ -46,6 +46,23 @@ iERR ion_test_open_file_writer(hWRITER *writer, FILE *out, BOOL is_binary) {
     iRETURN;
 }
 
+TEST_F(WriterTest, ResourcesNotLeakedOnWriteToTooSmallBuffer)
+{
+    hWRITER writer = NULL;
+    ION_WRITER_OPTIONS options = {
+        .output_as_binary = true
```

Comment:
It looks like the Windows build (via AppVeyor) [doesn't like](https://ci.appveyor.com/project/tgregg/ion-c-3akm7/builds/41820078/job/3tdu1mqxms1th8ew#L397) this syntax.